exercise_09/getDistanceMessageFromSumTo100.js

```diff
@@ -0,0 +1,40 @@
+import generateRandomNumberInRange from "../exercise_03/generateRandomNumberInRange.js";
+import getDistanceFromThreshold from "../exercise_01/getDistanceFromThreshold.js";
+
+/** 
+ * Write JS function "getDistanceMessageFromSumTo100" to generate a random number (in between 0-100) and sum it to given number
+  - if sum result exceeds 100, then return message "Sum with value N exceeds in N from number 100"
+  - otherwise, return message "Sum with value N is left in N from number 100"
+ * input: number (value) 
+ * output: string
+ */
+
+/**
+ * Depending on the sum result of a generated random number and a given number, it will return a different message
+ * @param {number} number
+ * @returns {string} message
+ */
+const getDistanceMessageFromSumTo100 = function (number) {
+  const randomNumber = generateRandomNumberInRange(100);
+  const sum = randomNumber + number;
+  const distanceNumber = getDistanceFromThreshold(sum, 100);
+
+  return sum > 100
```

Comment:
We defined a method to check wether a value is greater or not than a threshold.

FIX: use abstractions, not implementations.